systers_portal/meetup/views.py

```diff
@@ -1457,3 +1459,19 @@ def get(self, request, *args, **kwargs):
             # TODO: configure logging and log the unknown status
         return super(CancelMeetupLocationJoinRequestView, self).get(request, *args,
                                                                     **kwargs)
+
+
+@api_view(['POST'])
+def api_for_vms(request):
```

Comment:
Instead of function based views, please use class based views as it has higher capabilities.